src/gatherers/gather.js

```diff
@@ -47,6 +47,8 @@ class Gather {
 
   afterSecondReloadPageLoad(options) { }
 
+  afterThirdReloadPageLoad(options) { }
```

Comment:
I think we should try and roll this into the second page reload. That's only used for HTTP => HTTPS redirection atm, so there's no reason not to use that phase here.